tools/clang/include/clang/SPIRV/SpirvBuilder.h

```diff
@@ -443,10 +443,11 @@ class SpirvBuilder {
 
   /// \brief Creates an OpBitFieldInsert SPIR-V instruction for the given
   /// arguments.
-  SpirvBitFieldInsert *
-  createBitFieldInsert(QualType resultType, SpirvInstruction *base,
-                       SpirvInstruction *insert, SpirvInstruction *offset,
-                       SpirvInstruction *count, SourceLocation);
+  SpirvInstruction *createBitFieldInsert(QualType resultType,
+                                         SpirvInstruction *dst,
```

Comment:
Can we keep the names as base and insert to match the SPIR-V spec? I also think it's a bit more clear than src and dst, both here and for the emulated version.